src/globals.c

```diff
@@ -206,14 +210,39 @@ static int set_default_value_from_path(char **global, const char *path)
 		*c = '\0';
 	}
 
-	string_or_die(global, "%s", line);
+	string_or_die(contents, line);
 	ret = 0;
 fail:
 	fclose(file);
 	free(rel_path);
 	return ret;
 }
 
+int get_int_from_path(const char *abs_path) {
```

Comment:
What is the int we are getting with this function? Could be renamed to be a bit more clear.